modules/connection.js

```diff
@@ -0,0 +1,22 @@
+var mysql=require('mysql');
+var connection=mysql.createConnection({
+	host:'localhost',
+	user:'root',
+	password:'',
+	database:'meete'
+});
+connection.connect(function(err){
+	if(err){
+		var error={
+			status: 0,
+			message:"Error in Execution"
+		}
```

Comment:
it will show error message in postman console : 
status: 0,
message:"Error in Execution"